src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -1,56 +1,212 @@
 using Microsoft.Extensions.Logging;
+using NHS.ServiceInsights.Common;
+using NHS.ServiceInsights.Model;
 using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
```

Comment:
Don't think we need this using directive

Reply:
removed